forte_wrapper/spacy/spacy_processors.py

```diff
@@ -46,13 +47,28 @@ def set_up(self):
             download(self.lang_model)
```

Comment:
Improve the introduction, including what models we wrap, and the citation and project links to each of them.